tests/integration/preconfig-integration-test.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+set -euo pipefail
+
+# This script is expected to be called from the root folder of Connaisseur
+
+echo 'Preparing Connaisseur config...'
+yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' helm/values.yaml tests/integration/preconfig-update.yaml
+yq e '.policy +={"pattern": "docker.io/securesystemsengineering/connaisseur:helm-hook-*"} | .policy[4].pattern style="double"' -i helm/values.yaml
```

Comment:
What does this do? Would it be possible to integrate it into the `preconfig-update.yaml` file?

Reply:
this excludes the helm-hook from validation, since a new built is not signed and thus uninstallation would fail. If you write that into the `preconfig-update.yaml` we overwrite all other policies or would have to add all and at that point it is likely that test and actual preconfig diverge...